dev-db/influxdb/influxdb-1.2.4.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=6
+
+inherit user unpacker
+
+DESCRIPTION="Scalable datastore for metrics, events, and real-time analytics"
+HOMEPAGE="http://influxdb.com"
+SRC_URI="https://dl.influxdata.com/influxdb/releases/${PN}_${PV}_amd64.deb"
+
+RESTRICT="mirror"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+
+pkg_setup() {
+	enewgroup ${PN}
+	enewuser ${PN} -1 -1 "/var/lib/${PN}" ${PN}
+}
+
+src_unpack() {
+	mkdir -p ${WORKDIR}/${P}
+	cd ${WORKDIR}/${P}
+	unpack_deb ${A}
+}
+
+src_install() {
+	cp -Rp * "${D}"
```

Comment:
|| die

This seems to include a fair number of incorrectly installed files:
* docs in `/usr/share/doc/influxdb`,
* compressed manpages,
* `/usr/lib/influxdb/scripts` which seem to serve no explicit purpose in install -- yet includes a systemd service file which you are not installing.